acme/acme/client.py

```diff
@@ -602,16 +629,17 @@ def new_account(self, new_account):
         response = self._post(self.directory['newAccount'], new_account)
         # if account already exists
         if response.status_code == 200 and 'Location' in response.headers:
-            raise errors.ConflictError(response.headers.get('Location'))
+            raise errors.ConflictError(response.headers.get('Location', 'UNKOWN-LOCATION'))
```

Comment:
I had a suggestion for this [here](https://github.com/certbot/certbot/pull/9036#discussion_r714304830) (which I think is OK because the previous line checks if `Location` is in `response.headers`).

Reply:
Indeed we can use @bmw suggestion here. I close this comment.